api/src/main/resources/messages.properties

```diff
@@ -1,3 +1,5 @@
 ${project.parent.artifactId}.title=Legacy UI Module
 ${project.parent.artifactId}.manage=Manage module
 ${project.parent.artifactId}.patient.duplicateName.voided={0} is a duplicate name for the same patient and it's voided. Please restore the existing name instead of creating new name
+User.providerAccount=Provider Account
```

Comment:
I didn't understand how the prefix codes work. Can you please explain how I'm supposed to do it? And why?